tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    local_salt(salt): adds a salt to the seed provided by the pytest-rng fixture
+rng_salt = v0.41.1
```

Comment:
Should this be changed with each new PL release? If so, is this documented anywhere?

Reply:
Yes, this should be added to the release guideline. I added it as a TODO to [this PR](https://github.com/PennyLaneAI/guidance-docs/pull/65).

Reply:
We don't need a GitHub Action for that. WE can add a small script to [Ringo](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/.github/workflows/update_dev_version.yml) and make it do the work. All the permissions are already figured out and the version accessible. We just open this file as a dictionary in python and update the field.